src/utils/tawk.ts

```diff
@@ -11,13 +11,16 @@ export type TawkAPI = {
   visitor: Visitor;
   onLoad(): void;
   addTags(tags: UserRole[], callback?: () => void): void;
+  hideWidget(): void;
+  toggle(): void;
+  showWidget(): void;
 };
 
 function init() {
   if (process.env.NODE_ENV === 'development' || isSsr()) {
-    return;
+    // return;
```

Comment:
Let's keep the return.
It's rare we change things in the chat area and it's annoying to work on the project and keep getting the chat sounds and effects.

Reply:
ive done this

Reply:
ive done that